linux/net/rina/serdes.c

```diff
@@ -791,8 +791,6 @@ static struct pdu_ser * pdu_serialize_gfp(gfp_t                       flags,
             ASSERT(dup_chksum_is_ok(tmp));
         }
 
-        pdu_destroy(pdu);
-
```

Comment:
Why is this removed? pdu_serialize takes ownership of the pdu, converts it into a pdu_ser and destroys the pdu.